src/lib/Elf.cc

```diff
@@ -73,37 +77,38 @@ Elf::Elf(std::string path, char** imagePointer) {
    */
 
   // Seek to the byte representing the start of the header offset table.
-  uint64_t headerOffset;
-  file.read(reinterpret_cast<char*>(&headerOffset), sizeof(headerOffset));
+  // Holds the program header table's file offset in bytes.  If the file has no
+  // program header table, this member holds zero
+  uint64_t e_phoff;
+  file.read(reinterpret_cast<char*>(&e_phoff), sizeof(e_phoff));
 
   /**
-   * Starting 54th byte of the ELF Header a 16-bit value indicates
-   * the size of each entry in the ELF Program header. In the `elf64_hdr`
-   * struct this value maps to the member `Elf64_Half e_phentsize`. All
-   * header entries have the same size.
+   * Starting from the 54th byte of the ELF Header a 16-bit value indicates
+   * the the size in bytes of one entry in the
+   * file's program header table; all entries are the same
+   * size. In the `elf64_hdr`
+   * struct this value maps to the member `Elf64_Half e_phentsize`.
```

Comment:
There are a lot of references to elf64_hdr and now an Elf64_Half data type. Please link in the reference material for all this (similar to how we've done in the `createStack` function).

Reply:
I think this is already done on line 14 but could add again if you think it would be more clear